.github/workflows/pr.yaml

```diff
@@ -0,0 +1,79 @@
+name: run
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  stage_pr:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: 'read'
+      pull-requests: 'write'
+      id-token: 'write'
+
+    steps:
+    - uses: actions/checkout@v2
+      name: 'Check out Github repository'
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - run: cd web ; npm install ; npm run build
```

Comment:
These `;` should be `&&` just in case.

Reply:
Good call will fix.

Reply:
Fixed:
https://github.com/GoogleCloudPlatform/gcping/blob/add-ghaction/.github/workflows/pr.yaml#L22